### PR TITLE
Support Bio NER using stanza processor

### PR DESCRIPTION
**Pull request by Leolty**
This PR fixes asyml/ForteHealth/issues/19.

### Description of changes
I add NER configuration in stanza processor
with the following configuration:
`
config = {
  "processors":{
       "tokenize":"default",
       "pos":"defualt",
       "lemma":"default",
       "depparse":"default",
       "ner":"i2b2"
        },
  "lang": "en",
  "use_gpu": False,
}
`
You can get biological/ medical/ clinical EntityMention of your input data. (This is BioNER)

Also, you can feel free to change the model in the configuration, 
for example, with the following  configuration:
`
config = {
  "processors":{
       "tokenize":"default",
       "pos":"defualt",
       "lemma":"default",
       "depparse":"default",
       "ner":"ontonotes"
        },
  "lang": "en",
  "use_gpu": False,
}
`
You can get basic EntityMention of your input data. (This is NER) 

### Possible influences of this PR.
Support basic NER and Bio NER using stanza processor: all you need to do is modify the configuration according to your requirements.

### Test Conducted
All the test data are explicitly described in stanfordnlp_processor_test.py, and the test is successful.

### CODE REVIEW

**Comment by Leolty** [comment]
I suddenly found a problem with the naming of processors: it should be `StanfordNLPProcessor` instead of `Standfor**d**NLPProcessr`. I will add this issue after this PR is merged.

File: src/stanza/fortex/stanza/stanza_processor.py
Comment (Piyush13y) [inline src/stanza/fortex/stanza/stanza_processor.py:63]: Again, we can just use `self.processors` here right? Since it is being set when `self.set_up()` is called in L58.

**Review by Piyush13y — CHANGES_REQUESTED** [review]

**Comment by Leolty** [comment]
@Piyush13y  I have resolved all the changes you requested and change the styles of the test codes by using ddt. 

Is it inevatible to use hardcoding to check pos, lemma, dependency parsing, ner and bio ner?

I noticed that the tests in nltk and icd coding both use this kind of hardcodings.

**Comment by Piyush13y** [comment]
> @Piyush13y I have resolved all the changes you requested and change the styles of the test codes by using ddt.
> 
> Is it inevatible to use hardcoding to check pos, lemma, dependency parsing, ner and bio ner?
> 
> I noticed that the tests in nltk and icd coding both use this kind of hardcodings.

Unit tests do have the hardcoded component most times, since its generally the processing of a component that we are testing. Hence, we use pre determined results and compare the results with it to know its working in the intended manner. My comment was more about the way you were hardcoding, it wasnt needed to make a list of each token written explicitly when you could leverage `split() `method itself for this.

**Comment by Leolty** [comment]
Bad news is that build errors come one after another. 

Does the workflows run automatically or it will run after an approval from a maintainer? It seems like I always waited more than one day for the results of the workflows.

**Comment by Leolty** [comment]
@Piyush13y Could you plz review the requested changes. I have resolved all. XD

**Review by Piyush13y — APPROVED** [review]

**Pull request merged**
